### PR TITLE
cli tool to help lql authoring, watch folder, on edit json, lql print out entity result

### PR DESCRIPTION
**Pull request by araddon**
refs #4 

See README https://github.com/lytics/go-lytics/blob/4_watch/cmd/lytics/README.md

Watch a folder with lytics cli client, upon change of .lql or .json files upload the data in json with the query in .lql to the query/test endpoint to evaluate and show how it was evaluated.  Prints out json response of user profile to stdout upon change.

Allows rapid try/response on how queries/data will be interpreted.

### CODE REVIEW

File: cmd/lytics/main.go [hunk 1]
```diff
@@ -148,6 +161,8 @@ func (c *Cli) handleFunction(method string) (string, error) {
 	case "user":
 		result, err = c.getUsers(id)
 
+	case "watch":
+		c.watch()
 	default:
 		output = "Unknown Method"
 	}
```
Comment (ashyablok) [inline cmd/lytics/main.go:171]: I think at the bottom of this file there is a long string that get's printed out for `lytics --help` or if the user tries an incorrect command. It has a list of methods, which would be nice to have the watch command on.

**Review by markhayden — APPROVED** [review]
this is going to be super helpful and hopefully cut down on rebuilds quite a bit. definitely need to get dave and dan using it as soon as possible.

File: query.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package lytics
```
Comment (ashyablok) [inline query.go:1]: Probably not super important if we're looking to get this out ASAP, but usually we have tests for these. Mark or I can add them later too.

**Review by ashyablok — APPROVED** [review]
I tested it and it worked for me! Mark knows more about the command line tool and just LQL in general, so he will probably have a more insightful review.

File: cmd/lytics/main.go (hunk 1 above)
Comment (araddon) [reply cmd/lytics/main.go:171]: good call, did that.

File: query.go (hunk 2 above)
Comment (araddon) [reply query.go:1]: yes, i really should, i feel guilty

**Pull request merged**